source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,145 @@
+
+The content summarises the architecture by stating what happens when a user:
+
+- visits a page on GOV.UK
+- publishes content to GOV.UK
+
+As this is for both a developer and non-developer audience, the content contains [definitions](#definitions) at the end.
+
+## A user visits a page on GOV.UK
+
+This content will go through 3 scenarios where a user:
+
+- visits the GOV.UK homepage
+- visits a GOV.UK webpage
+- searches GOV.UK using the search page
+
+### A user visits the GOV.UK homepage
+
+The user visits https://www.gov.uk or https://gov.uk to get to the GOV.UK hpmepage.
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
+
+[Fastly](https://www.fastly.com/) is a cloud computing services provider and content delivery network (CDN).
+
+GOV.UK uses the Fastly CDN to handle the majority of requests to GOV.UK. This reduces the load on GOV.UK Origin by around 70%.
+
+GOV.UK Origin is a stack of computers in the AWS cloud, and its entry point is a load balancer. Origin routes requests to the machine running the application that can respond to the request.
+
+Fastly also provides edge nodes or servers that are closer to our users, particularly those outside the UK.
+
+If Fastly has the homepage cached, Fastly returns the homepage to the user.
+
+If Fastly does not have the homepage cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+The request from Router is then proxied (or forwarded) to an application called Frontend.
+
+[Frontend](https://github.com/alphagov/frontend) is a public-facing app that serves the majority of documents on the /government part of GOV.UK. These government pages are fetched from the Content Store by the appropriate application. Frontend is a replacement for the public-facing parts of the Whitehall application.
+
+Frontend produces the homepage as an HTML response and sends that response back to Router.
+
+Router sends the response to Fastly, which caches the response for 30 minutes. This means that Fastly can handle any subsequent requests for the homepage during this time, without needing to query Origin at all.
+
+Finally, Fastly sends the response back to the user.
+
+### A user visits a GOV.UK webpage
+
+The user searches for a GOV.UK webpage, and selects the webpage link in the search results. For example, this [news page about changes to travel between Nigeria and England](https://www.gov.uk/government/news/uk-changes-travel-rules-for-travellers-from-nigeria-to-england).
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
```

Comment:
A spanner in the works: subsequent requests to GOV.UK (such as this one for the news article) won't go through the whole DNS lookup again. It's already completed that rather time-expensive query, and doesn't need to do so again for several hours. The browser will just cache the IP address from the first time, and use that for all subsequent requests to GOV.UK.

In hindsight, I think it may be best to just omit the DNS info altogether. It's a tech thing that applies to any website (not just GOV.UK) and - as we can see here! - it can get a bit complicated.

I think I'd simplify to something along the lines of "The user's browser sends the request to the Fastly content delivery network (CDN)."